apiv2/schema/schema.yaml

```diff
@@ -699,25 +677,36 @@ classes:
       plural:
         tag: plural
         value: AnnotationAuthors
-    description: Author of an annotation
+    description: Metadata for an annotation's authors
     from_schema: cdp-dataset-config
     mixins:
     - IDMixin
     - AuthorEntityMixin
     attributes:
       annotation:
         name: annotation
-        description: Metadata about an annotation for a run
+        description: Reference to the annotation this author contributed to
         from_schema: cdp-dataset-config
         inverse: Annotation.authors
         range: Annotation
+    slot_usage:
```

Comment:
We're inheriting a bunch of fields from the `AuthorEntityMixin` and this `slot_usage` block lets us override properties of the inherited fields